developer/src/kmc-package/test/fixtures/example.qaa.sencoten/example.qaa.sencoten.model.kmp.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
The fixtures here are copied from the kmc-model fixtures, so that they are accessible for the kmc-package test (they used to be shared files for both tests).

We _could_ create a common test fixtures folder, but it hardly seems worthwhile.